Statistics/Distributions/Deterministic.cs

```diff
@@ -42,22 +42,6 @@ public class Deterministic : IDistribution
         public Deterministic(double x)
         {
             Value = x;
-            BuildFromProperties();
-        }
-        public void BuildFromProperties()
```

Comment:
I am not sure I understood the purpose of BuildFromProperties. Why did we have it, and then why remove it?

Reply:
build from properties really exists for LP3 and truncated distributions when we change min or max, we need to make sure that the probability range gets updated, or when we change the lp3 stuff we have to trigger the range being set to a finite range for some reason.